### DISCUSSION TITLE
Media window always on top - Can I disable this feature?

### DISCUSSION THREAD

**Discussion by heymenshan** [sircharlo/meeting-media-manager#2590] [Q&A] [answered]
I'm using M3 both on my Mac and a Windows computer. But on my Windows, the Media window is always on top. Can I disable this feature?

**Comment by sircharlo** [comment]
In the new, revised version of the app that is coming out next month, the media window is always on top only when fullscreen. Does this behavior sound like it's acceptable? Ie:
- Fullscreen media window: Always on top
- Windowed/resizeable media window: Not always on top

**Comment by heymenshan** [comment, marked as answer]
Oh, thanks for letting me know. I can start using the app on Windows then.Sent from my iPhoneOn Sep 9, 2024, at 10:53 PM, Charles Olivier Savignac ***@***.***> wrote:﻿
In the new, revised version of the app that is coming out next month, the media window is always on top only when fullscreen. Does this behavior sound like it's acceptable? Ie:

Fullscreen media window: Always on top
Windowed/resizeable media window: Not always on top
